pkg/performanceprofile/controller/performanceprofile/components/consts.go

```diff
@@ -12,6 +12,8 @@ const (
 	ComponentNamePrefix = "performance"
 	// MachineConfigRoleLabelKey is the label key to use as label and in MachineConfigSelector of MCP which targets the performance profile
 	MachineConfigRoleLabelKey = "machineconfiguration.openshift.io/role"
+	// MachineConfigPoolRoleLabelPrefix
+	MachineConfigPoolRoleLabelPrefix = "pools.operator.machineconfiguration.openshift.io/"
```

Comment:
I would probably move this to the create function. Just to keep it close to the other hardcoded names.

Reply:
Sounds good, done